aip/aog/3010.md

```diff
@@ -0,0 +1,55 @@
+---
+aip:
+  id: 3010
+  state: draft
+  created: 2020-01-14
+  updated: 2020-01-14
+  type: guidance
+  scope: aog
+js:
+  - /assets/js/graphviz/viz.js
+  - /assets/js/graphviz/lite.render.js
+  - /assets/js/aip/aip-graphviz.js
+permalink: /aog/3010
+---
+
+# Actions on Google Vertical Integration Webhook Format
+
+If an action is part of a vertical program that uses Google Fulfillment, it
```

Comment:
I know you mentioned to me via email that Google Fulfillment isn't a public concept, and I think it's still okay to mention it here (not great, but I don't have a better idea), but calling this out to make sure other reviewers have a chance to weigh in.

Reply:
I think we need some way to avoid mentioning Google Fulfillment. Maybe instead, we can say here:

"part of a vertical program that requires a webhook for Google to integrate with your action."

Reply:
Rephrased here and other places in this doc.

Reply:
LGTM